CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing guidelines
+
+## Pull Request Checklist
+
+Before sending your pull requests, make sure you followed this list.
+
+- Read [contributing guidelines](CONTRIBUTING.md).
```

Comment:
This could be confusing. Maybe specify that the reader is already looking at CONTRIBUTING.md by just saying something like "Read this file completely."?

Reply:
Oh yeah I saw that, I put the wrong link in, my bad